addons/mass_mailing_sms/tests/common.py

```diff
@@ -35,12 +35,12 @@ def assertSMSStatistics(self, recipients_info, mailing, records, check_sms=True)
           :param recipients_info: list[{
             'partner': res.partner record (may be empty),
             'number': number used for notification (may be empty, computed based on partner),
-            'state': outgoing / sent / ignored / exception / opened (sent by default),
+            'state': outgoing / sent / canceled / exception / opened (sent by default),
             'record: linked record,
             'failure_type': optional: sms_number_missing / sms_number_format / sms_credit / sms_server
             }, { ... }]
         """
-        traces = self.env['mailing.trace'].search([
+        traces = self.env['mail.notification'].search([
```

Comment:
rename `traces` to `notifications` ?

And below your `notif` can become `recipient_notifications` ?